src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -71,16 +75,16 @@ public boolean extractQuestionDetails(
             Map<String, String[]> requestParameters,
             FeedbackQuestionType questionType) {
         String numOfRubricChoicesString = HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_NUM_COLS);
+                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_NUM_COLS);
         String numOfRubricSubQuestionsString = HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                                     Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_NUM_ROWS);
+                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_NUM_ROWS);
```

Comment:
Were the changes to these indentations absolutely necessary?

Reply:
> Were the changes to these indentations absolutely necessary?

Hi @madanalogy,
No. The indentation changes are not necessary. I just checked without these changes and there was no stylecheck violation. Please ignore these indentation changes.